Makefile

```diff
@@ -188,7 +188,7 @@ endif
 
 .PHONY: install
 install: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
-	$(KUSTOMIZE) build config/crd | kubectl apply -f -
+	$(KUSTOMIZE) build config/crd | kubectl replace --force -f -
```

Comment:
Why force is needed? that also removes CRDs and custom resources

Reply:
Because I've seen it fail otherwise during development complaining about some incompatibility.